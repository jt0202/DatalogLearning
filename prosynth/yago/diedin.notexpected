august_zingel	akron_ohio
virginia_de_medici	ho_chi_minh_city
bill_hopkin	jersey_city_new_jersey
frederick_ix_of_denmark	aberdeen
nicolae_pleşiţă	geneva
