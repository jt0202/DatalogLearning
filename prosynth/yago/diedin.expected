siva_selliah	colombo
fumiko_enchi	tokyo
sergey_yustinovich_bagotsky	moscow
prince_francis_xavier_of_saxony	dresden
prince_axel_of_denmark	copenhagen
eugen_barbu	bucharest
john_george_i	lt_Elector_of_SaxonyDresdengt
daniel_adam_z_veleslavína	prague
oskar_wolkerstorfer	linz
florence_harding	marion_ohio
frederick_christian	lt_Elector_of_SaxonyDresdengt
feodor_i_of_russia	moscow
werner_van_den_valckert	amsterdam
daniel_rodney	lewes_delaware
eugèneanatole_demarçay	paris
albert_g_schmedeman	madison_wisconsin
frithjof_sælen_writer	bergen
albert_tocco	chicago
edouard_wah	portauprince
rose_of_viterbo	viterbo
gia_carangi	philadelphia
eugen_filotti	bucharest
joseph_warder	jeffersonville_indiana
alfonso_iv_deste	lt_Duke_of_ModenaModenagt
adrienmarie_legendre	paris
cesare_arzelà	santo_stefano_di_magra
mantarō_kubota	tokyo
eberhard_rodt	munich
abel_gance	paris
antónio_augusto_soares_de_passos	porto
felix_of_nola	nola
clifford_walker	monroe_georgia
liverus_hull	westfield_massachusetts
vere_ponsonby	lt_9th_Earl_of_BessboroughLondongt
alice_cogswell	hartford_connecticut
jean_bernard_physician	paris
muhammad_ali_jinnah	karachi
roy_mcelroy	auckland
benito_alazraki	mexico_city
tzeni_karezi	athens
elizabeth_siddal	london
sybil_arundale	london
lucien	lt_Lord_of_MonacoMonacogt
reg_plummer_rugby_player	newport
jerzy_jurandot	warsaw
guido_gozzano	turin
isabel_burton	london
pope_theodore_ii	rome
walter_bestmann	lübeck
joão_do_canto_e_castro	lisbon
